packages/android_alarm_manager/README.md

```diff
@@ -29,6 +30,7 @@ void printHello() {
 
 main() async {
   final int helloAlarmID = 0;
+  await AndroidAlarmManager.initialize();
```

Comment:
Good catch! The call to `AndroidAlarmManager.periodic` below would have be run before the `AndroidAlarmManager.initialize` call is executed. Thanks!